### ISSUE TITLE
Proper HDMI Output

### ISSUE THREAD

**Issue by KakarottoCake** [mpv-android/mpv-android#728]
### Describe the wanted behaviour of the new feature and potential usecases for it here.

When connected to HDMI, output the video to the display connected for accurate screen resolution of desired video. This will allow for devices that do not have conventional screen resolutions to output video without black bars to modern TVs.

### Alternative behavior of the wanted feature
N/A

VLC does this, but their implementation for subs is questionable for anime which is why I would love to see this on MPV

**Comment by KakarottoCake** [comment]
@sfan5 VLC stops displaying the video on the device screen and ouputs it to the TV instead. For insurance, I have a Samsung tablet that has a strange aspect ratio and the HDMI outputs black bars on the TV. VLC corrects this by outputting ONLY the video to the TV which in turn also accounts for the Black Bars.

**Comment by sfan5** [comment]
https://github.com/mpv-android/mpv-android/issues/943#issuecomment-2160293186

**Comment by KakarottoCake** [comment]
Considering it's been a little over a year now, has there been any progress on this?

**Comment by sfan5** [comment]
No. Progress only happens if I work on this. And I have limited time as well as other priorities.

**Comment by simon-weber** [comment]
I'm not at all familiar with android dev but I can take a stab at this. It's an important feature for display glasses which may otherwise have limited options to turn off the phone display.

**Comment by simon-weber** [comment]
Opened https://github.com/mpv-android/mpv-android/pull/1326.
